scm/driver/github/webhook.go

```diff
@@ -147,6 +149,16 @@ func (s *webhookService) parsePullRequestHook(data []byte) (scm.Webhook, error)
 	return dst, nil
 }
 
+func (s *webhookService) parsePullRequestReviewCommentHook(data []byte) (scm.Webhook, error) {
+	src := new(pullRequestReviewCommentHook)
+	err := json.Unmarshal(data, src)
+	if err != nil {
+		return nil, err
+	}
+	dst := convertPullRequestReviewCommentHook(src)
+	return dst, nil
```

Comment:
this should set the action to one of `scm.ActionCreate`, `scm.ActionUpdate`, `scm.ActionDelete`